docs/troubleshooting/vm.md

```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 5
+sidebar_label: VM
+title: "VM"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/troubleshooting/vm"/>
```

Comment:
do we need to change the URL here? (and version v1.2's URL)

Reply:
From the `Deploy Preview`, it seems the version value  `v1.1` here does not affect. But please @LucasSaintarbor @jillian-maroket  double check, thanks.